phpstan.neon

```diff
@@ -1,12 +1,13 @@
 parameters:
-	customRulesetUsed: true
-	reportUnmatchedIgnoredErrors: false
-	# Ignore phpstan-drupal extension's rules.
-	ignoreErrors:
-		- '#\Drupal calls should be avoided in classes, use dependency injection instead#'
-		- '#Plugin definitions cannot be altered.#'
-		- '#Missing cache backend declaration for performance.#'
-		- '#Plugin manager has cache backend specified but does not declare cache tags.#'
+  level: 1
+  customRulesetUsed: true
+  reportUnmatchedIgnoredErrors: false
+  # Ignore phpstan-drupal extension's rules.
+  ignoreErrors:
+    - '#\Drupal calls should be avoided in classes, use dependency injection instead#'
+    - '#Plugin definitions cannot be altered.#'
+    - '#Missing cache backend declaration for performance.#'
+    - '#Plugin manager has cache backend specified but does not declare cache tags.#'
 #includes:
 #	- vendor/mglaman/phpstan-drupal/extension.neon
 #	- vendor/phpstan/phpstan-deprecation-rules/rules.neon
```

Comment:
Thanks.  This change [clearly works](https://github.com/localgovdrupal/localgov_project/actions/runs/6315995866/job/17149743370?pr=131#step:4:39).  But do we want commented out config here?

Reply:
Thanks @Adnan-cds good point it was already there so I've removed the commented out code

Reply:
Sorry, didn't realize it was there already.  The copy from the 2.x branch had it uncommented which confused me.  I am okay with this change.  But merging it will means all test runs are going to fail until a massive code clean up is complete.  So I will leave this with Finn @finnlewis  :)